shell/src/test/java/org/apache/kafka/shell/MetadataNodeManagerTest.java

```diff
@@ -336,4 +403,43 @@ public void testProducerIdsRecord() {
             11000 + "",
             metadataNodeManager.getData().root().directory("producerIds").file("nextBlockStartId").contents());
     }
+
+    @Test
+    public void testAccessControlEntryRecordAndRemoveAccessControlEntryRecord() {
+        AccessControlEntryRecord record1 = new AccessControlEntryRecord()
+            .setId(Uuid.fromString("GcaQDl2UTsCNs1p9s37XkQ"))
```

Comment:
nit: since this is just for test, we can use `Uuid.ZERO_UUID` for testing, to avoid random strings here.

Reply:
This is a little overkill here, we are using random strings in a lot of test cases and it won't reduce the code quality, and `Uuid.ZERO_UUID` may have some special meaning in different cases.